corgidrp/l3_to_l4.py

```diff
@@ -860,13 +860,14 @@ def northup(input_dataset,use_wcs=True,rot_center='im_center',new_center=None):
     return processed_dataset
 
 
-def determine_wave_zeropoint(input_dataset, template_dataset = None, xcent_guess = None, ycent_guess = None, bb_nb_dx = None, bb_nb_dy = None, return_all = False):
+def determine_wave_zeropoint(input_dataset, spec_filter_offset, template_dataset = None, xcent_guess = None, ycent_guess = None, bb_nb_dx = None, bb_nb_dy = None, return_all = False):
```

Comment:
we need to modify the l3_to_l4*spec.json recipe templates to specify that we need to pass in and automatically select a SpecFilterOffset file, and make sure the two spec_l3_to_l4 e2e still run.